gatsby-ssr.js

```diff
@@ -1,17 +1,18 @@
+// gatsby-ssr.js
 import React from "react";
-
-const OutsetaScriptComponent = ({ site }) => {
-
-return (
-    <script
-      key="fathom-script"
-      src="https://cdn.outseta.com/outseta.min.js"
-    ></script>
+const OutsetaScriptComponent = () => {
```

Comment:
1. Looks like gatsby is not reading my `const o_options` variable the way I want. 👇 😺 
Uncaught Error: [domain] is a required option to initialize the Outseta script.

2. But gatsby IS reading my `script` tag
look 👇 😸 👍  
outseta.min.js:2 

Here is the screenshot:
![Geoff-Uncaught Error-  domain  is a required option to initialize the Outseta script -3 3](https://user-images.githubusercontent.com/7642493/144410516-34ec76bc-1315-4632-8beb-05cc84ed7a14.png)

Reply:
I am still getting this error in my Console


**Uncaught Error:** [domain] is a required option to initialize the Outseta script.

outseta.min.js:2 

in this commit [e0ee2b9](https://github.com/olavea/Rubys-TimeShip/pull/13/commits/e0ee2b9273cde2c131d3882b38809d2fb1806d14)